### DISCUSSION TITLE
Welcome to Haraka Discussions!

### DISCUSSION THREAD

**Discussion by msimerson** [haraka/Haraka#3044] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by atonb117** [comment]
Just found out about you from the Loomio server install docs. Looking forward to learn more this exciting project! #JSforTheWin!
